src/main/java/org/umssdiplo/automationv01/core/utils/CommonEvents.java

```diff
@@ -20,7 +20,11 @@
      */
     public static void setInputField(WebElement webElement, String content) {
```

Comment:
we already check it, please slipt this method in two methods, according your requeriments.